Doc/library/os.path.rst

```diff
@@ -351,22 +356,32 @@ the :mod:`glob` module.)
 
 
 .. function:: join(path, *paths)
-
+    
    Join one or more path segments intelligently.  The return value is the
    concatenation of *path* and all members of *\*paths*, with exactly one
    directory separator following each non-empty part, except the last. That is,
    the result will only end in a separator if the last part is either empty or
    ends in a separator. If a segment is an absolute path (which on Windows
    requires both a drive and a root), then all previous segments are ignored and
    joining continues from the absolute path segment.
-
+   
+   Examples::
```

Comment:
I would create also create a separate for "If a segment is an absolute path". and end the paragraph by "For instance::" and show the examples.